Cargo.toml

```diff
@@ -149,7 +149,6 @@ gpu-descriptor = "0.3"
 bit-set = "0.8"
 gpu-allocator = { version = "0.27", default-features = false }
 range-alloc = "0.1"
-hassle-rs = "0.11.0"
```

Comment:
For future reference, the only reason why `hassle-rs` cannot use `windows-rs` as bindings is because the `windows-rs` maintainers refuse to allow basic Linux support. DirectxShaderConverter works on Linux via the same bindings.

(Either way the abstraction provided by `hassle-rs` is far from ideal)

Reply:
I found it more straightforward to just use the bindings directly since we have to deal with them for D3D12 anyway.